src/database/accounts.js

```diff
@@ -51,29 +51,16 @@ module.exports = {
     },
 
     /**
-     * Search for a user by name
+     * Search for a user by any field
      */
-    search: function (name, fields, callback) {
-        /* This bit allows it to accept varargs
```

Comment:
Was destructuring unavailable when you wrote that bit? I've been looking for good excuses to make use of it, but never seem to find them.
```js
function destructuredVarArgs({ name, fields, callback }){
  fields = typeof fields === 'undefined' ? ['name'] : fields;
}
```